README.md

```diff
@@ -65,6 +66,7 @@ systems).
   * `cwd` - **[optional]** The working directory for the command. E.g. what `.` resolves to.
   * `env` - **[optional]** An array of environment variables and their values to set
   * `errorMatch` - **[optional]** A regular expression to match output to a file, row and col. See [Error matching](#error-match) for details.
+  * `buildViewTimeout` - **[optional]** Interval in milliseconds to dismiss build view
```

Comment:
You should add some thing like " ... upon successful build.'. This probably also needs some clarifications on how it works in conjunction with the configuration variable `panelVisibility`, which can be `Toggle`, `Keep Visible` or `Show on Error`.